File: ts/wam/2808.ts
export const WamChatFolderOpen = 2808 as const

export interface ChatFolderOpenEvent {
  /** field 1, wire `folder_type` */
  folderType?: string
  /** field 2, wire `activity_indicator_count` */
  activityIndicatorCount?: number
  /** field 3, wire `has_important_messages` */
  hasImportantMessages?: boolean
}

File: go/wam/2808.go
const WamChatFolderOpen = 2808 // channel: regular

type ChatFolderOpenEvent struct {
	FolderType string `wam:"folder_type"` // field 1
	ActivityIndicatorCount int64 `wam:"activity_indicator_count"` // field 2
	HasImportantMessages bool `wam:"has_important_messages"` // field 3
}